web/src/components/tasks/business-formation/review/section/ReviewSection.tsx

```diff
@@ -33,9 +37,9 @@ export const ReviewSection: React.FC<PropsWithChildren<Props>> = ({
 
   return (
     <>
-      <div className="flex space-between">
-        <div className="maxw-mobile-lg margin-bottom-2">
-          <h2 className="h3-styling">{header}</h2>
+      <div className={marginOverride ? `flex space-between ${marginOverride}` : `flex space-between`}>
+        <div className={sectionType === "MAIN" ? "maxw-mobile-lg margin-bottom-2" : "maxw-mobile-lg"}>
```

Comment:
This comment is a note for the future, because if you follow my suggestion to make these two components, this ternary will go away altogether.

BUT:

If you do find yourself writing a className ternary like this and the one above in the future, then I recommend reducing duplication by only inserting the part that matters. Like so:
```
className={`maxw-mobile-lg ${sectionType === "MAIN" ? "margin-bottom-2" : ""}`}
```

This makes it more future-proof if we add more classes.

Imagine if you had like four more shared class names and the one that changed. In your style, it'd look like:

```
className={sectionType === "MAIN" ? "maxw-mobile-lg class-1 class-2 class-3 class-4 margin-bottom-2" : "maxw-mobile-lg class-1 class-2 class-3 class-4"}
```

in my suggestion, it'd look like this:

```
className={`maxw-mobile-lg class-1 class-2 class-3 class-4 ${sectionType === "MAIN" ? "margin-bottom-2" : ""}`}
```